### ISSUE TITLE
Change the play bar

### ISSUE THREAD

**Issue by timhot** [mcuz/cacophony-audiotagger#6]
Change the play bar (the bit with the moving dot, to something like you get with Audacity that shows a volume plot and perhaps a frequency plot below it.  Give the user the ability to select a region with mouse click/drag - and then select tag.

**Comment by marknikora** [comment]
Yes looks possible with web audio api and found js library that would suit development of this
https://wavesurfer-js.org/
